website/react-magma-docs/src/pages/api/theme.mdx

```diff
@@ -109,6 +146,746 @@ export function Example() {
 }
 ```
 
+## Springfield Donut
+
+Who doesn't like a good one?
+
+```tsx
+import React from 'react';
+import {
+  AppBar,
+  Button,
+  ButtonColor,
+  ButtonGroup,
+  Card,
+  CardBody,
+  CardHeading,
+  Combobox,
+  Drawer,
+  magma,
+  Radio,
+  RadioGroup,
+  Spacer,
+} from 'react-magma-dom';
+
+import styled from '../../theme/styled';
+
+const springfieldDonut = Object.assign({}, magma, {
+  colors: Object.assign({}, magma.colors, {
+    danger: '#f3772b',
+    danger600: '#b94d1c',
+    marketing: '#1a85e0',
+    primary: '#3493f1',
+    primary600: '#215fb3',
+    neutral: '#7b361e',
+    secondary500: '#42e093',
+    secondary600: '#43fd94',
+  }),
+
+  bodyFont: '"Varela Round", sans-serif',
+
+  typeScale: Object.assign({}, magma.typeScale, {
+    size02: {
+      fontSize: '16px',
+      letterSpacing: '-1px',
+    },
+    size03: {
+      fontSize: '20px',
+      letterSpacing: '-1px',
+    },
+  }),
+
+  appBar: {
+    backgroundColor: '#d792cb',
+    textColor: '#ebf88b',
+    height: '80px',
+    padding: '0 30px',
+  },
+});
```

Comment:
Instead of a partial stylesheet override (like the examples above), I think this example should override the [ENTIRE](https://github.com/cengage/react-magma/blob/dev/packages/react-magma-dom/src/theme/magma.ts#L492-L793) stylesheet and change almost every value to showcase how that can be done.